beacon_node/store/src/hot_cold_store.rs

```diff
@@ -1748,6 +1914,24 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         self.blob_info.read_recursive().clone()
     }
 
+    /// Initialize the `DataColumnInfo` when starting from genesis or a checkpoint.
+    pub fn init_data_column_info(&self, anchor_slot: Slot) -> Result<KeyValueStoreOp, Error> {
+        let oldest_data_column_slot = self.spec.deneb_fork_epoch.map(|fork_epoch| {
```

Comment:
should this be pectra fork epoch

Reply:
Good point - currently we're testing peerdas on top of deneb, so we've been using deneb in a few places, but this should be `eip7594_fork_epoch` and we can change it to electra later once we rebase PeerDAS on `electra`.